bin/make-manifest

```diff
@@ -1,19 +1,23 @@
 #!/usr/bin/env node
 
-var program = require('commander')
-var makeManifest = require('../')
+const program = require('commander');
+const _set = require('lodash/set');
```

Comment:
Maybe we can use directly `lodash.set` if we are not planning to use more utils from Lodash? [See](  https://www.npmjs.com/package/lodash.set)

Reply:
Interesting. I was investigating how to do it in vanilla JS, but it's easier to just import what @UlisesGascon is proposing

Reply:
Great suggestion! Done!